src/docker.ts

```diff
@@ -52,6 +56,8 @@ const saveDockerImages = async (): Promise<void> => {
           "will be saved."
       );
       const newImagesArgs = newImages.join(" ");
+      info("newImagesArgs");
+      info(newImagesArgs);
```

Comment:
Was this left over from debugging? `execBashCommand` already logs the command passed to it, which contains this string.